runtime/reference/cli/compiler.md

```diff
@@ -3,14 +3,22 @@ title: "`deno compile`, standalone executables"
 oldUrl: /runtime/manual/tools/compiler/
 ---
 
-`deno compile [--output <OUT>] <SRC>` will compile the script into a
-self-contained executable.
+`deno compile` allows you to create self-contained executables from a TypeScript
+or JavaScript file.
 
-```console
-> deno compile https://docs.deno.com/examples/welcome.ts
+This feature allows distribution of a Deno application to systems that do not
+have Deno installed. Under the hood, `deno compile` bundles a slimmed down
+version of the Deno runtime along with your JavaScript or TypeScript code.
+
+```sh
+deno compile https://docs.deno.com/examples/welcome.ts
```

Comment:
Change it to use `jsr` instead please, maybe `@std/http/file-server`? (remember to add proper perm flags)